packaging/ubuntu-14.04/snapd.prerm

```diff
@@ -36,6 +36,9 @@ if [ -d /run/systemd/system ]; then
         done
     fi
 
-    systemctl stop snapd.autoimport.service snapd.socket snapd.service
+    # the wrapper makes sure that the service is stopped
+    systemctl_stop snapd.autoimport.service || true
```

Comment:
Why || true? I think stop can be called multiple times (and doesn't report error if already stopped)?